packages/react-magma-docs/src/pages/design/card.mdx

```diff
@@ -7,20 +7,17 @@ order: 1
 import { Link } from 'gatsby';
 import './design-guidelines-styles.css';
 import DocsHeading from '../../components/docs-heading';
-import { Paragraph } from 'react-magma-dom';
+import LeadParagraph from '../../components/lead-paragraph';
 
 <DocsHeading to="/api/card/" type="design">
   Card
 </DocsHeading>
-
-<Paragraph visualStyle="bodyLarge">
+<LeadParagraph>
   Cards provide a flexible container with multiple variants and options.
-</Paragraph>
-
-<span class="horizontal-spacer"></span>
+</LeadParagraph>
 
 <figure>
-  <img src="../../images/cards/card-intro-example.png" alt=" " />
+  <img src="../../images/cards/card-intro-example.png" alt="" />
```

Comment:
don't we want a space in the `alt` here for screen readers?

Reply:
If memory serves me correctly, I believe so.

Reply:
Good catch!  Will change